src/inspector/worker_agent.cc

```diff
@@ -0,0 +1,155 @@
+#include "worker_agent.h"
+
+#include "main_thread_interface.h"
+#include "worker_inspector.h"
+
+namespace node {
+namespace inspector {
+namespace protocol {
+
+class NodeWorkers
+    : public std::enable_shared_from_this<NodeWorkers> {
+ public:
+  explicit NodeWorkers(std::weak_ptr<NodeWorker::Frontend> frontend,
+                      std::shared_ptr<MainThreadHandle> thread)
+                      : frontend_(frontend), thread_(thread) {}
+  void WorkerCreated(const std::string& title,
+                 const std::string& url,
+                 bool waiting,
+                 std::shared_ptr<MainThreadHandle> target);
```

Comment:
Style nit: Here and above, the parameters aren’t aligned (and we generally only indent the initializers starting with `:` by 2 spaces.)

Side note: Running `make format-cpp` (preceded by `make format-cpp-build`, if necessary) would make sure you don’t have to worry about style nits anymore.